Src/Base/AMReX_FabArray.H

```diff
@@ -2506,6 +2510,16 @@ FabArray<FAB>::setDomainBndry (value_type val,
     }
 }
 
+template <class FAB>
+template <class F, std::enable_if_t<IsBaseFab<F>::value,int> FOO>
+void
+FabArray<FAB>::capacityOfFabs(LayoutData<std::size_t>& mem) {
+    // Should we have assertions here? What should be asserted?
```

Comment:
It would be good to assert that `mem` and `*this` use the same BoxArray and DistributionMap. You can use the `CellEqual` method for BoxArray and the `==` operator for the DistributionMap.